### DISCUSSION TITLE
JLC Files

### DISCUSSION THREAD

**Discussion by mlinton** [bendecaf/xfm2-pcbs#13] [General]
Hey just wondering if you ended up saving this project on JLCPCB in a way that we can add it to a cart to order?  Want to give the extended board a try.

**Comment by bendecaf** [comment]
https://github.com/bzeiss/xfm2-pcbs/blob/main/extended/gerber/xfm2_xva1_extended_board_v1.40.zip

For assembly:
https://github.com/bzeiss/xfm2-pcbs/blob/main/extended/gerber/xfm2_xva1_extended_board_v1.40-top-bom.csv
https://github.com/bzeiss/xfm2-pcbs/blob/main/extended/gerber/xfm2_xva1_extended_board_v1.40-top-pos.csv

It's possible that there is still an error on the pcb assembly with one of the ic's. I recall that one footprint that was slightly too big and I think I have never corrected this. Not a problem for soldering, but a problem for the machine based assembly. So a few smd components still have to be soldered by hand if you decide to have jlcpcb assemble things as well. The through-hole components have to be soldered by hand anway though.

Be aware that the power regulation is undersized for the Eurorack 12V connector. I don't recommend using the eurorack connector at this point.

**Reply by mlinton** [reply]
Well, just got mine delivered from JLC, without U8 or U10 soldered.  Got the parts from Aliexpress, now to test my TSSOP hand soldering skills!  Which component is over/undersized?

I'm going to power it through the barrel jack so no worries there.  Thanks again for sharing.

**Comment by mlinton** [comment]
Finally got around to soldering this up one of the partially assembled JCL boards.  I had to solder U10 and U8.  Inspected everything and tested for continuity before assembling and it checked out.

Got the A7 loaded with the XVA1 software, patches and tunings, when using xload I wasnt' able to use the * command to get the MIDI channel or set it. - strange.

Anyway powered it up, all looks good and I see the green power LED and MIDI activity light blinking, but I can't get any sound out of it.  From either the balanced jacks or from the 3.5mm out on the sparkfun analog board.  Any ideas on how to troubleshoot?

**Reply by mlinton** [reply]
update:  when I connect the board with the barrel jack, then the audio doesn't work, but if I connect the CMOD-A7 with the micro-usb cable it then works (and I can set the MIDI channel, etc).  Not sure how that would affect it.

**Reply by bendecaf** [reply]
This is weird. I'm pretty sure that I had this working with the barrel jack. I'll give this a try when I find the time for it.

**Reply by mlinton** [reply]
Thanks man - its working now from the USB, but it's likely something I did soldering one of the components (although most of the power circuit was straight from JLC).
